From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Data Plus <46394572+shapotkin@users.noreply.github.com>
Date: Mon, 13 May 2019 10:14:32 +0300
Subject: [ARCHEOLOGY] Create add-nanopi-neo-core.patch

> X-Git-Archeology: > recovered message: > [ nanopi-neo-core ] Add support for eMMC within u-boot
> X-Git-Archeology: - Revision 1951a89e35482ddbc9f124079b64bcc2f2e8d98f: https://github.com/armbian/build/commit/1951a89e35482ddbc9f124079b64bcc2f2e8d98f
> X-Git-Archeology:   Date: Mon, 13 May 2019 10:14:32 +0300
> X-Git-Archeology:   From: Data Plus <46394572+shapotkin@users.noreply.github.com>
> X-Git-Archeology:   Subject: Create add-nanopi-neo-core.patch
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 58d80414057b57359d60fa91293b82ac734efc06: https://github.com/armbian/build/commit/58d80414057b57359d60fa91293b82ac734efc06
> X-Git-Archeology:   Date: Tue, 13 Jun 2023 17:03:41 +0200
> X-Git-Archeology:   From: Ricardo Pardini <ricardo@pardini.net>
> X-Git-Archeology:   Subject: `uboot`: introduce `u-boot-sunxi-v2022.04` for Allwinner boards using 22.04 u-boot (`nanopineo`)
> X-Git-Archeology:
---
 arch/arm/dts/sun8i-h3-nanopi-neo.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/dts/sun8i-h3-nanopi-neo.dts b/arch/arm/dts/sun8i-h3-nanopi-neo.dts
index 9f33f6fae5..4a56e6a950 100644
--- a/arch/arm/dts/sun8i-h3-nanopi-neo.dts
+++ b/arch/arm/dts/sun8i-h3-nanopi-neo.dts
@@ -58,6 +58,16 @@
 	status = "okay";
 };
 
+&mmc2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&mmc2_8bit_pins>;
+	vmmc-supply = <&reg_vcc3v3>;
+	bus-width = <8>;
+	non-removable;
+	cap-mmc-hw-reset;
+	status = "okay";
+};
+
 &ohci0 {
 	status = "okay";
 };
-- 
Armbian

